file / libmagic: missing dependency on libzstd

Maintainer: @ratkaj, last updated by @VolkerChristian 
Environment: mvebu / Linksys WRT1900ACv1 / OpenWrt git main branch pulled yesterday

Description:
I get this error trying to build an image with "file" utility.
```
 /bin/mkdir -p '/build/openwrta/build_dir/target-arm_cortex-a9+vfpv3_musl_eabi/file-5.44/ipkg-install/usr/bin'
  /bin/sh ../libtool   --mode=install /build/openwrta/staging_dir/host/bin/install -c file '/build/openwrta/build_dir/target-arm_cortex-a9+vfpv3_musl_eabi/file-5.44/ipkg-install/usr/bin'
libtool: warning: 'libmagic.la' has not been installed in '/usr/lib'
libtool: install: /build/openwrta/staging_dir/host/bin/install -c .libs/file /build/openwrta/build_dir/target-arm_cortex-a9+vfpv3_musl_eabi/file-5.44/ipkg-install/usr/bin/file
 /bin/mkdir -p '/build/openwrta/build_dir/target-arm_cortex-a9+vfpv3_musl_eabi/file-5.44/ipkg-install/usr/include'
 /build/openwrta/staging_dir/host/bin/install -c -m 644 magic.h '/build/openwrta/build_dir/target-arm_cortex-a9+vfpv3_musl_eabi/file-5.44/ipkg-install/usr/include'
make[6]: Leaving directory '/build/openwrta/build_dir/target-arm_cortex-a9+vfpv3_musl_eabi/file-5.44/src'
make[5]: Leaving directory '/build/openwrta/build_dir/target-arm_cortex-a9+vfpv3_musl_eabi/file-5.44/src'
make[4]: Leaving directory '/build/openwrta/build_dir/target-arm_cortex-a9+vfpv3_musl_eabi/file-5.44/src'
Package libmagic is missing dependencies for the following libraries:
libzstd.so.1
make[3]: *** [Makefile:117: /build/openwrta/bin/packages/arm_cortex-a9_vfpv3/packages/libmagic_5.44-1_arm_cortex-a9_vfpv3.ipk] Error 1
```

Thanks.

--- a/libs/file/Makefile
+++ b/libs/file/Makefile
@@ -56,8 +56,10 @@ CONFIGURE_ARGS += \
 	--enable-xzlib \
 	--enable-zlib \
 	--disable-libseccomp \
+	--disable-lzlib \
 	--disable-rpath \
 	--disable-warnings \
+	--disable-zstdlib \
 	--without-pic
 
 MAKE_PATH := src

